src/filters/mno-currency/mno-currency.filter.coffee

```diff
@@ -4,7 +4,7 @@
 # $filter('mnoCurrency')(amount,currency,[true|false])
 #
 angular.module('impac.filters.mno-currency', []).filter('mnoCurrency', ($filter) ->
-  (amount, currency='', ISOmode=true) ->
+  (amount, currency='', ISOmode=true, decimal) ->
```

Comment:
This gives the ability to set how many decimal places you would like. Before this change currency would always be 2 decimal places (which makes sense as a default), but the mockups had no decimal places. Where would you like this to be documented? Just a comment above?